editor/filesystem_dock.cpp

```diff
@@ -3086,6 +3086,8 @@ void FileSystemDock::_folder_color_index_pressed(int p_index, PopupMenu *p_menu)
 
 	_update_tree(get_uncollapsed_paths());
 	_update_file_list(true);
+
+	emit_signal(SNAME("folder_color_changed"), selected, chosen_color_name ? folder_colors[chosen_color_name] : Color(), chosen_color_name ? chosen_color_name : "default");
```

Comment:
Now I don't get this signal. Do I see it right that you send both color and its name? The arguments you send are not even used. What are they for?